src/pendingactions.js

```diff
@@ -52,7 +52,7 @@ import CKEditorError from '@ckeditor/ckeditor5-utils/src/ckeditorerror';
  *
  * @extends module:core/plugin~Plugin
```

Comment:
```suggestion
 * @extends module:core/contextplugin~ContextPlugin
```

Reply:
Not sure why the first space is shown by GitHub's diff as removed because it's there in the suggestion ;|